src/main/resources/application-custom.yaml

```diff
@@ -268,7 +268,7 @@ hapi:
     # classes listed here will be fetched from the Spring context when combined with 'custom-bean-packages',
     # or will be instantiated via reflection using an no-arg contructor; then registered with the server
     #custom-interceptor-classes:
-    custom-interceptor-classes: ca.uhn.fhir.jpa.starter.interceptor.CorrelationIdInterceptor
+    custom-interceptor-classes: ca.uhn.fhir.jpa.starter.interceptor.LoggerInterceptor
```

Comment:
This change needs to be coordinated with a change in many other repos. 

Other repos, like our *-r4 and *-r4-terminology, have their own configuration files that refer to the interceptor. Changing the name here means that they all need to change too.